apiserver/facades/client/application/deployrepository.go

```diff
@@ -93,10 +93,10 @@ func NewDeployFromRepositoryAPI(
 func (api *DeployFromRepositoryAPI) DeployFromRepository(ctx context.Context, arg params.DeployFromRepositoryArg) (params.DeployFromRepositoryInfo, []*params.PendingResourceUpload, []error) {
```

Comment:
Might be worth putting a comment on this function to point out that the PendingResourceUpload argument is deprecated and just there for compatibility (or it can be removed and the facade version bumped if that is not the case).